package/MDAnalysis/lib/mdamath.py

```diff
@@ -279,7 +279,7 @@ def _is_contiguous(atomgroup, atom):
     return True
 
 
-def make_whole(atomgroup, reference_atom=None):
```

Comment:
So you've changed how this function can be called, which isn't usually allowed, ie maybe someone was using the `reference_atom` functionality and we can't take that away.